oracle-plugin/src/e2e-test/java/io.cdap.plugin/common.stepsdesign/TestSetupHooks.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.common.stepsdesign;
+
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.OracleClient;
+import io.cucumber.java.Before;
+
+import java.sql.SQLException;
+
+/**
+ * Oracle test hooks.
+ */
+public class TestSetupHooks {
+
+  @Before(order = 1)
+  public static void overrideCredentials() {
+    String username = System.getenv("ORACLE_USERNAME");
+    if (username != null && !username.isEmpty()) {
+      PluginPropertyUtils.addPluginProp("ORACLE_USERNAME", username);
+    }
+    String password = System.getenv("ORACLE_PASSWORD");
+    if (password != null && !password.isEmpty()) {
+      PluginPropertyUtils.addPluginProp("ORACLE_PASSWORD", password);
+    }
+    String host = System.getenv("ORACLE_HOST");
+    if (host != null && !host.isEmpty()) {
+      PluginPropertyUtils.addPluginProp("ORACLE_HOST", host);
+    }
+    String port = System.getenv("ORACLE_PORT");
+    if (port != null && !port.isEmpty()) {
+      PluginPropertyUtils.addPluginProp("ORACLE_PORT", port);
+    }
+  }
```

Comment:
why is this hook needed?

Reply:
Removed the hook. Using these credentials directly from the environment now.